### DISCUSSION TITLE
Relate SiteVisit to SoilSite/LandSurveySite

### DISCUSSION THREAD

**Discussion by abhritchie** [ANZSoilData/def-au-schema-json#17] [General]
`SiteVisit` has relationships to `SoilSites` and `LandSurveySites` but I can't find a explicitly declared relationship in the other direction. Only `Treatment` has a `relatedActivity` property.

Given our JSON Schema profile assumes we start with a `SoilSite` as the root element, being able to access a visit from there is valuable - this is where data information comes from.

Also, if we do this, is `relatedProject` necessary on a site if the `SiteVisit` also has `relatedProject`? The project is time dependent, especially if sites are revisited.

**Comment by abhritchie** [comment]
`SiteVisit` also has a `relatedActivity` so we can add the `Sampling` activity here to bring in all the samples taken at the site at a given time.

**Reply by dr-shorthair** [reply]
Yes. That was the whole point. The activity-oriented view point is based on the site visit.
